### PR TITLE
Add RefreshableUpdates

### PR DESCRIPTION
**Pull request by TimWSpence**
Add `RefreshableUpdate` subtype that supports getting a stream of all discrete updates

### CODE REVIEW

File: core/src/main/scala/com/permutive/refreshable/Refreshable.scala [hunk 1]
```diff
@@ -53,196 +55,73 @@ trait Refreshable[F[_], A] { outer =>
...
+
+      override def get: G[CachedValue[A]] = fk(self.get)
+      override def cancel: G[Boolean] = fk(self.cancel)
+      override def restart: G[Boolean] = fk(self.restart)
+    }
+
 }
 
-object Refreshable {
+trait RefreshableUpdates[F[_], A] extends Refreshable[F, A] { self =>
```
Comment (janstenpickle) [inline core/src/main/scala/com/permutive/refreshable/Refreshable.scala:75]: I kind of like putting the subclass inside the object so we have `Refreshable.Updates`.

**Review by janstenpickle — COMMENTED** [review]
Looks good, just a question about the placement of the new trait

File: core/src/main/scala/com/permutive/refreshable/Refreshable.scala (hunk 1 above)
Comment (TimWSpence) [reply core/src/main/scala/com/permutive/refreshable/Refreshable.scala:75]: sure 👍

**Review by janstenpickle — APPROVED** [review]
LGTM!

**Pull request merged**